recipe/bld.bat

```diff
@@ -0,0 +1,47 @@
+mkdir build
+cd build
+
+cmake ^
+  -G "%CMAKE_GENERATOR%" ^
+  -D CMAKE_BUILD_TYPE:STRING=RELEASE ^
+  -D CMAKE_INSTALL_PREFIX:PATH=%PREFIX% ^

```

Comment:
Use `conda-forge`'s convention of `%LIBRARY_PREFIX%` instead of just `%PREFIX%`.